tests/bluetooth/ctrl_user_ext/testcase.yaml

```diff
@@ -0,0 +1,7 @@
+common:
+  build_only: true
+  tags: bluetooth
+tests:
+  bluetooth.ctrl_user_ext.test:
+    platform_whitelist: qemu_cortex_m3 nrf52840_pca10056 nrf52_pca10040
```

Comment:
Any reason to not build for nrf52_bsim but building for all these instead?

Reply:
I dont think you can build the BT controller on qemu_cortex_m3

Reply:
good suggestion, building for nrf52_bsim now